package.json

```diff
@@ -43,22 +47,25 @@
     "lodash.throttle": "^4.1.1",
     "lodash.topairs": "^4.3.0",
     "micro-cors": "^0.1.1",
-    "next": "^9.1.6",
     "node-cache": "^5.1.0",
     "perfect-scrollbar": "^1.4.0",
+    "prismjs": "^1.19.0",
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
     "react-collapse": "^5.0.1",
     "react-collapsible": "^2.6.2",
     "react-dom": "^16.12.0",
     "react-ga": "^2.7.0",
+    "react-helmet": "^5.2.1",
     "react-markdown": "^4.2.2",
     "react-popover": "^0.5.10",
     "react-scroll": "^1.7.13",
     "react-slick": "^0.25.2",
     "react-syntax-highlighter": "^11.0.2",
```

Comment:
do we need this? is there a way to cleanup the dependencies?

Reply:
We can safely remove `react-syntax-highlighter` here, yes, as well as `react-markdown` and some other libraries.

Reply:
Ok. Removed `react-syntax-highlighter` but didn't remove `react-markdown`. I forgot that it is used in Tooltip. Created ticket for removal https://github.com/iterative/dvc.org/issues/1045

Reply:
what about other libraries?

Reply:
`unist-util-visit` is still used, but now on server instead of client. Same with `isomorphic-fetch` (we still using it for API). Need to check others in more details, but at the first glance we still need them.